cms_form/static/src/js/master_slave.js

```diff
@@ -53,6 +50,24 @@ odoo.define('cms_form.master_slave', function (require) {
           });
         });
       },
+      eval_master_slave: function (master_value, values, handler, slave) {
+        if (typeof(values) === 'string') {
+          if (pyeval.py_eval(values, this.eval_master_slave_context())) {
+              handler(slave);
+          }
+        }
+        else if (_.contains(values, master_value)) {
+          handler(slave);
+        }
+      },
+      eval_master_slave_context: function () {
+        var result = {form: {}};
```

Comment:
`var result = {'form': this.$el.serializeArray()}`
BTW  probably you can use this in #101 too.